projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,37 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: inline-block;
+    border-color: var(--tui-base-02);
+    color: var(--tui-primary);
+    border-radius: 50%;
+    width: 1rem;
+    height: 1rem;
+    outline: 0.1875rem solid var(--tui-base-01);
+    box-shadow: 0 0 0.4375rem rgba(0, 0, 0, 0.3);
```

Comment:
I wonder if because of outline, rather than border, box-shadow should also have 0.1875rem as 4th argument so it "starts" from the edge of the outline, not from the body of the compass.

Reply:
fixed